troposphere/static/js/components/projects/detail/resources/ImageActionButtons.react.js

```diff
@@ -0,0 +1,48 @@
+define(function (require) {
+
+  var React = require('react/addons'),
+    Backbone = require('backbone'),
+    Button = require('./Button.react'),
+    actions = require('actions'),
+    modals = require('modals');
```

Comment:
Just for the _new_ files. If the file/module was present before - I have converted it.